templates/pgbackrest.conf.j2

```diff
@@ -4,6 +4,7 @@ lock-path=/tmp
 log-path={{ log_path }}
 repo1-retention-full-type=time
 repo1-retention-full={{ retention_full }}
+repo1-retention-history=365
```

Comment:
Are we wiping backups older then 1 year by default? It can be unacceptable for list of companies we know.

Reply:
What happens here is: Whenever pgbackrest creates a backup, it will also create an associated manifest file that contains some information about the backup itself, WAL size, etc. When a backup is expired, its manifest file does not expire automatically, but is kept as a "history" of backups/WAL size growth.

This configuration specifies that, whenever a backup is expired, its associated manifest file will only expire a year later. 

Valid non-expired backups will not have their manifest files deleted, so if the user sets 10 years retention, the manifest files will stay for 11 years.

365 is a rather typical value for this config (from as far as I could see online and on the docs).